contracts/abstracts/Base.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import { Ownable } from "./Ownable.sol";
+import { Asset } from "../libraries/Asset.sol";
+import { Constant } from "../libraries/Constant.sol";
+
+/// @title Base Contract
+/// @author imToken Labs
+abstract contract Base is Ownable {
```

Comment:
Maybe it could have a more meaningful name, for example `TokenManager` or `TokenManagement`.

Reply:
Renamed as `AdminManagement` in 80d671b